Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -393,6 +393,34 @@ extension KeyInfo {
         if let key = (try? privateKey()) { return key }
         return try? delegatePrivateKey()
     }
+
+    func pushNotificationSigningKey(completion: @escaping (Result<PrivateKey?, Error>) -> ()) {
```

Comment:
Looks like we're not using this anywhere except int the async wrapper version. Can we get rid of this by rewriting the async method or at least annotate it with 
```
@available(*, deprecated, renamed: "pushNotificationSigningKey()")
```
So it is not used anywhere? Or we could make it private?

Reply:
yes, I will see that this function is not exposed